src/timedate/timedatectl.c

```diff
@@ -124,10 +124,10 @@ static void print_status_info(const StatusInfo *i) {
         else
                 tzset();
 
-        printf("                       Time zone: %s (%s)\n"
-               "       System clock synchronized: %s\n"
-               "systemd-timesyncd.service active: %s\n"
-               "                 RTC in local TZ: %s\n",
+        printf("                        Time zone: %s (%s)\n"
+               "        System clock synchronized: %s\n"
+               "systemd-timesyncd.service enabled: %s\n"
```

Comment:
This change looks correct, even if doesn't fix everything. This value comes from https://github.com/sourcejedi/systemd/blob/828553eb0d2d4e05eb074ba9cb9aba2ac6179e05/src/timedate/timedated.c#L179 `use_ntp = STR_IN_SET(s, "enabled", "enabled-runtime")`, so "enabled" is certainly more appropriate than "active".